gateway/src/apicast/policy/scope_check/scope_check.lua

```diff
@@ -0,0 +1,88 @@
+local policy = require('apicast.policy')
+local _M = policy.new('Scope Check Policy')
+
+local ngx_variable = require ('apicast.policy.ngx_variable')
+
+local ipairs = ipairs
+local format = string.format
+local gsub = ngx.re.gsub
+local match = ngx.re.match
+
+local TemplateString = require 'apicast.template_string'
+
+local new = _M.new
+
+local function build_templates(scopes)
```

Comment:
Why using Liquid?

It is not exposed to users and it would be much simpler to just use a table to index roles and then look them up.

Reply:
@mikz Yes, you're right. Fixed.